manifests/status.pp

```diff
@@ -25,7 +25,7 @@
       mysql_user { "${status_user}@${status_allow}":
         ensure        => 'present',
         password_hash => mysql_password($status_password),
-        require       => [File['/root/.my.cnf'],Service['mysqld']]
```

Comment:
Why was the dependeny on `File['/root/.my.cnf']` removed?

Reply:
Because there is no such file resource in the catalogue.

Reply:
The test failing sets `create_root_my_cnf` to false. The status manifest still expects there to be a .my.cnf  looks like a missing condition to me. I will investigate it further

Reply:
Implemented a workaround.